src/libraries/System.Runtime/tests/System/EnumTests.cs

```diff
@@ -262,120 +262,230 @@ public static void Parse_Invalid(Type enumType, string value, bool ignoreCase, T
             }
         }
 
-        public static IEnumerable<object[]> GetName_TestData()
+        public static IEnumerable<object[]> GetName_SByteEnum_TestData()
         {
-            // SByte
-            yield return new object[] { typeof(SByteEnum), SByteEnum.Min, "Min" };
-            yield return new object[] { typeof(SByteEnum), SByteEnum.One, "One" };
-            yield return new object[] { typeof(SByteEnum), SByteEnum.Two, "Two" };
-            yield return new object[] { typeof(SByteEnum), SByteEnum.Max, "Max" };
-            yield return new object[] { typeof(SByteEnum), sbyte.MinValue, "Min" };
-            yield return new object[] { typeof(SByteEnum), (sbyte)1, "One" };
-            yield return new object[] { typeof(SByteEnum), (sbyte)2, "Two" };
-            yield return new object[] { typeof(SByteEnum), sbyte.MaxValue, "Max" };
-            yield return new object[] { typeof(SByteEnum), (sbyte)3, null };
+            yield return new object[] { SByteEnum.Min, "Min" };
+            yield return new object[] { SByteEnum.One, "One" };
+            yield return new object[] { SByteEnum.Two, "Two" };
+            yield return new object[] { SByteEnum.Max, "Max" };
+            yield return new object[] { sbyte.MinValue, "Min" };
+            yield return new object[] { (sbyte)1, "One" };
+            yield return new object[] { (sbyte)2, "Two" };
+            yield return new object[] { sbyte.MaxValue, "Max" };
+            yield return new object[] { (sbyte)3, null };
+        }
```

Comment:
Given how you split things up and that this method is only used with one test, how about making all of these InlineData instead?  Seems simpler.